draft-js-emoji-plugin/src/index.js

```diff
@@ -92,18 +121,21 @@ export default (config = {}) => {
   // breaking change. 1px of an increased padding can break a whole layout.
   const {
     theme = defaultTheme,
-    positionSuggestions = defaultPositionSuggestions,
     imagePath = defaultImagePath,
     imageType = defaultImageType,
     allowImageCache,
-    priorityList,
+    positionSuggestions = defaultPositionSuggestions,
+    suggestionsPriorityList,
```

Comment:
@bashkos can you change it back to `priorityList`. With these changes I like your naming more, but I think it's an unnecessary breaking change that would cause more confusion than it helps.